portfolio/src/main/webapp/index.html

```diff
@@ -10,8 +10,7 @@
     <div id="content">
       <h1>Jonathan's Portfolio</h1>
       <h2>Welcome to my Portfolio!</h2>
-      <button onclick="addMessage()">Fetch Greeting</button>
-      <h1><span id="message-container"></span></h1>
+      <button><a href="write-message.html">Write a Comment</a></button>
```

Comment:
The recommended way to create a button that links to a page (with javascript) is:

```html
<input type="button" onclick="location.href='write-message.html';" value="Write a comment" />
```